src/EmbedIO/ExceptionHandler.cs

```diff
@@ -120,7 +121,7 @@ public static Task HtmlResponse(IHttpContext context, Exception exception)
                     }
                 });
 
-        internal static async Task Handle(string logSource, IHttpContext context, Exception exception, ExceptionHandlerCallback? handler)
+        internal static async Task Handle(string logSource, IHttpContext context, Exception exception, ExceptionHandlerCallback? handler, HttpExceptionHandlerCallback? httpHandler)
```

Comment:
What if I also want to handle exceptions thrown by the HTTP exception handler? One more `catch` block, one more handler... And what about HTTP exceptions thrown by _that_ handler? And so on, and so forth.

Reply:
OK we have clarified that on Slack. 👍